src/types/line.js

```diff
@@ -190,9 +201,44 @@ export default class LineAnnotation extends Element {
 }
 
 LineAnnotation.id = 'lineAnnotation';
+
+const arrowHeadsDefaults = {
+  backgroundColor: undefined,
+  backgroundShadowColor: undefined,
+  borderColor: undefined,
+  borderDash: undefined,
+  borderDashOffset: undefined,
+  borderShadowColor: undefined,
+  borderWidth: undefined,
+  enabled: undefined,
+  fill: undefined,
+  length: undefined,
+  shadowBlur: undefined,
+  shadowOffsetX: undefined,
+  shadowOffsetY: undefined,
+  width: undefined
+};
+
 LineAnnotation.defaults = {
   adjustScaleRange: true,
-  backgroundShadowColor: 'transparent',
+  arrowHeads: {
+    backgroundColor: undefined,
```

Comment:
Probably works fine without the `undefined` values at this level. But common defaults would be preferred at this level (and when it should be distinct by default from line defaults).

For example do we prefer arrows to have borderDash by default when `defaults.elements.lineAnnotation.borderDash = [3, 3]`?

Reply:
In my opinion yes, because is the line the main object and should define the undefined options for the arrows heads.
Let me know if you disagree.

Reply:
agreed, I think you should then remove everything else but `start` and `end` from `arrowHeads`. Options can be given at that level, but we don't have any defaults there.